packages/kbn-alerts-grouping/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-browser",
   "id": "@kbn/alerts-grouping",
-  "owner": "@elastic/response-ops"
-}
+  "owner": [
+    "@elastic/response-ops"
+  ],
+  "group": "observability",
+  "visibility": "private"
```

Comment:
The platform owns the alerting grouping package and is intended to be used by all solutions. At the moment, it is only used by observability, but that will change in the future.

Reply:
How far off is that future? We're trying to minimise platform surface area, so we tend to prefer categorising packages in this situation as _solution-specific_ (see [thread](https://elastic.slack.com/archives/C07TCKTA22E/p1731581785919069)).

Reply:
Ok, we can leave it as it is atm and change it when the time comes. Thanks!